liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_interview_columns-001.sql

```diff
@@ -0,0 +1,16 @@
+IF NOT EXISTS (SELECT 1
```

Comment:
It looks like we have a difference in child table numbering between Observation child tables and Investigation child tables. @sveselev the current numbering would align with the Observation child table numbering- is that good to go?

Reply:
I like 022- for all interview related tables